### ISSUE TITLE
As a disaster coordinator, I can email individual volunteers, or groups of volunteers

### ISSUE THREAD

**Issue by BillWagner** [HTBox/crisischeckin#11]
This is how disaster coordinators get information to volunteers.

**Comment by tditiecher** [comment]
Is **disaster** coordinator equal to **humanitarian** coordinator (as mentioned on https://github.com/HTBox/crisischeckin/wiki/2.0-Domain-Model) ?

**Comment by BillWagner** [comment]
@tditiecher Yes, those terms are used interchangeably throughout the user community.
